platforms/core-runtime/service-provider/src/main/java/org/gradle/internal/service/scopes/GradleModuleServices.java

```diff
@@ -26,61 +26,33 @@
  */
 @ServiceScope(Scope.Global.class)
 public interface GradleModuleServices extends ServiceRegistrationProvider {
+
     /**
-     * Called once per process, to register any globally scoped services. These services are reused across builds in the same process.
-     * The services are closed when the process finishes.
-     *
-     * <p>Global services are visible to all other services.</p>
-     *
```

Comment:
Given that at this level the definition of each scope is very nuanced, it is better to have keep the documentation in a single place, aka directly on the `Scope` classes.

Reply:
> Given that at this level the definition of each scope is very nuanced, it is better to have keep the documentation in a single place, aka directly on the `Scope` classes.

I'd still keep a brief line here and below, something along the lines: `Called to register services in {@link Scope.Global}.`, without going into much details about what the scope means.

Reply:
Done